save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/controller/InitializeAgentsController.kt

```diff
@@ -26,10 +28,17 @@ class InitializeAgentsController {
     /**
      * @param execution
      * @return OK if everything went fine.
+     * @throws ResponseStatusException
      */
     @PostMapping("/initializeAgents")
     fun initialize(@RequestBody execution: Execution): Status {
         // TODO initialization
+        if (execution.status != ExecutionStatus.PENDING) {
+            throw ResponseStatusException(
+                HttpStatus.INTERNAL_SERVER_ERROR,
```

Comment:
I believe it should be error 4xx, because the requester has sent the incorrect execution

Reply:
Fixed